torch/fx/experimental/symbolic_shapes.py

```diff
@@ -655,9 +665,7 @@ def binary_magic_impl(self, other):
     def unary_magic_impl(self):
         op = method_to_operator(method)
         if SYM_FUNCTION_MODE:
-            r = _handle_sym_dispatch(op, (wrap_node(self),), {})
-            assert isinstance(r, SymTypes), type(r)
```

Comment:
Are there legitimately any unary operations on symbolic ints/floats that can return constants 🤔 (aka the justification for removing this assert)

Reply:
In this case no; I removed it mostly for uniformity (because I'm intentionally weakning the invariant)